about.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Holi 2023</title>
+  <link rel="stylesheet" href="./css/about.css">
+  <link rel="stylesheet" href="./css/index.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <div class="nav-bar">
+        <a href="index.html"><img src="./images/Logo.png" alt="holi-logo" class="logo"></a>
+        <button class="hambug-menu">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+        <ul class="mobile-menu">
+          <li><a class="menuItem" href="index.html">Home</a></li>
+          <li><a class="menuItem" href="about.html">About</a></li>
+          <li><a class="menuItem" href="#">Programs</a></li>
+          <li><a class="menuItem" href="#">Gallery</a></li>
+          <li><a class="menuItem" href="#">Come Celebrate</a></li>
+        </ul>
+
+        <ul class="nav-links">
+          <li class="links"><a class="alinks" href="about.html">About</a></li>
+          <li class="links"><a class="alinks" href="#">Programs</a></li>
+          <li class="links"><a class="alinks" href="#">Gallery</a></li>
+          <li class="links"><a class="alinks borderLink" href="#">Come Celebrate</a></li>
+        </ul>
+      </div>
+    </nav>
+
+    <section class="aboutHeadContent">
+      <h2 class="aboutHeaderTitle">"Have a Happy Holi!"</h2>
+      <h1 class="aboutSubHeader">HOLI - "Festival of Colors" or the "Festival of Love"</h1>
```

Comment:
- [ ] For accessibility purposes, kindly ensure the `<h1>` tag always comes before` h2` tag because heading tags have a hierarchy from h1-h6 which helps screen readers and browsers understand the priority you are given individual sections of your app. 

To lay emphasis or the most important heading of your app, use h1 and ensure you only use a single h1 throughout your website. This will help people easily find your website when they search for your heading text online 💯

Reply:
@vickymarz Thank you, I fixed it